src/test/specs/generic/TestUtils.spec.js

```diff
@@ -0,0 +1,16 @@
+class TestUtils {
+    /**
+     * Helper method to create an address object from a sequence of nibbles.
+     * @param {int[]} nibbles array of 40 nibbles (= 20 bytes)
```

Comment:
JavaScript doesn't have an integer type. `number[]` would work as a type annotation for `nibbles`, however, we prefer the notation `Array.<number>` to be consistent with other generic types.